Library/Homebrew/software_spec.rb

```diff
@@ -107,6 +108,10 @@ def resource(name, klass = Resource, &block)
     end
   end
 
+  def get_resource
+    @resource
+  end
```

Comment:
Sorry for the noise! I completely overlooked the existing `resource` method, but given how widely used it is, my comment feels a bit silly in retrospect. Thanks for checking nonetheless and if we don't need it anymore, that's even better.

However, should we find out we still need it, I think it could be called `url_resource` or something similar (to make it clear that it is not some random resource but the one associated with the primary `url` statement of the formula).